sdcm/sct_config.py

```diff
@@ -1550,6 +1550,10 @@ class SCTConfiguration(dict):
 
         dict(name="kafka_connectors", env="SCT_KAFKA_CONNECTORS", type=str_or_list_or_eval,
              help="configuration for setup up kafka connectors"),
+
+        dict(name="enable_force_gossip_topology_changes", env="SCT_ENABLE_FORCE_GOSSIP_TOPOLOGY_CHANGES", type=boolean,
```

Comment:
this param is looking like global one while it is only used in upgrade tests - misleading.

Btw. Can't we just add `force_gossip_topology_changes` to `append_scylla_yaml`? Now when it is used as dict and merged with other `append_scylla_yaml` params, maybe we don't need this one?

Reply:
We need this parameter to control different upgrade scenarios and trigger raft topology upgrade procedure